### PR TITLE
Add uvisersal method for screenshots

### PR DESCRIPTION
**Pull request by 0I0u3uk**
Added universal method which take value in application.properties and do screenshot.
Please, show my comment.

### CODE REVIEW

File: src/main/java/ru/sbtqa/tag/pagefactory/support/ScreenShooter.java [hunk 1]
```diff
@@ -48,4 +49,21 @@
         }
     }
 
+    /**
+     * Takes screenshot as indicated in application.properties
+     *
+     * @return screenshot in byte array
+     */
+    @Attachment(type = "image/png", value = "Screenshot")
```
Comment (0I0u3uk) [inline src/main/java/ru/sbtqa/tag/pagefactory/support/ScreenShooter.java]: Should be this annotation or will be work without this?

**Comment by ChuckSonarNorris** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [OnFailureScheduler.java#L4](https://github.com/sbtqa/page-factory/blob/757861c355b2cf8c3d7999aa909471776d1ef1f9/src/main/java/ru/sbtqa/tag/pagefactory/support/OnFailureScheduler.java#L4): Remove this unused import 'ru.sbtqa.tag.qautils.properties.Props'. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/main/java/ru/sbtqa/tag/pagefactory/support/ScreenShooter.java [hunk 2]
```diff
@@ -48,4 +48,26 @@
...
+     */
+    public static byte[] take() {
+        String screenshotStrategy = Props.get("screenshot.strategy", "raw");
+
+        byte[] screenshot;
+
+        switch (screenshotStrategy) {
+            case "driver":
+                screenshot = ScreenShooter.takeWithDriver();
+                ParamsHelper.addAttachment(screenshot, "Screenshot by Driver", Type.PNG);
```
Comment (kosteman) [inline src/main/java/ru/sbtqa/tag/pagefactory/support/ScreenShooter.java]: I think, that the main purpose of this class is taking screenshots, thats why we shouldn't do attaching to report here.

Comment (0I0u3uk) [inline src/main/java/ru/sbtqa/tag/pagefactory/support/ScreenShooter.java]: Hmmm, I think you're right, I will fix it

**Comment by 0I0u3uk** [comment]
https://github.com/sbtqa/docs/wiki/Screenshots-and-video-recording-tools 
doc updated

**Pull request merged**